Traceback (most recent call last):
  File "main.py", line 2, in <module>
    os.system("ls", "-l")
TypeError: system() takes exactly 1 argument (2 given)
